lighthouse-core/report/report-generator.js

```diff
@@ -261,14 +262,16 @@ class ReportGenerator {
     });
 
     const template = Handlebars.compile(this.getReportTemplate());
+    reportContext = reportContext || 'extension'; // could be: devtools, extension
```

Comment:
it should, but I can imagine someone using generateHTML on their own, and not setting the second argument. 
would you rather this go forward as undefined or we throw?